src/SonarScanner.MSBuild.PreProcessor/AnalysisConfigGenerator.cs

```diff
@@ -56,6 +56,7 @@ public static class AnalysisConfigGenerator
             SonarBinDir = buildSettings.SonarBinDirectory,
             SonarScannerWorkingDirectory = buildSettings.SonarScannerWorkingDirectory,
             SourcesDirectory = buildSettings.SourcesDirectory,
+            JavaExePath = localSettings.JavaExePath,
```

Comment:
Can you keep the TODO comment we had before? Or open a GitHub ticket and add a link to it here.